scripts/l10n.ts

```diff
@@ -432,10 +436,13 @@ async function run(argv: yargs.ArgumentsCamelCase<HandlerArgv>) {
           missingKeys: [],
         };
       }
-      const lines = rawCode.split('\n');
+      const lines = codeWithNewTranslations.split('\n');
```

Comment:
Using `rawCode` did not pick the `missingKeys` count correctly if the keys were updated during the same `l10n` run. So we had to run `l10n` twice to generate the correct keys' count in `data.json`.
Using transformed code fixed it.
(See 2nd commit and reverted commit for reference)